sdk/nodejs/Makefile

```diff
@@ -73,22 +73,23 @@ unit_tests:: $(TEST_ALL_DEPS)
 test_auto:: $(TEST_ALL_DEPS)
 	yarn run nyc -s mocha --timeout 300000 'bin/tests/automation/**/*.spec.js'
 
-TSC_SUPPORTED_VERSIONS = ~3.7.3 ^3 ^4
```

Comment:
I don't think ts3.7.3 works with pulumi, so I think this should be upgraded to 3.8